pkg/reconciler/revision/reconcile_resources.go

```diff
@@ -136,10 +159,21 @@ func (c *Reconciler) reconcileKPA(ctx context.Context, rev *v1alpha1.Revision) e
 	logger := logging.FromContext(ctx)
 	logger.Info("Reconciling KPA:", kpaName)
 
+	// TODO(vagababov): this part can be removed in 0.8
```

Comment:
How can it be removed, we don't transition folks to a deployment with the new name?